src/commands/addServerNamespaceToWorkspace.ts

```diff
@@ -80,82 +81,49 @@ export async function addServerNamespaceToWorkspace(): Promise<void> {
     return;
   }
   const { serverName, namespace } = picks;
-  let uri = vscode.Uri.parse(`isfs://${serverName}:${namespace}/`);
-  const api = new AtelierAPI(uri);
-  let project: string;
-  // Ask the user if they would like to use a project
-  const useProject = await vscode.window.showQuickPick(["Yes", "No"], {
-    placeHolder: "Use this folder to show a project's contents?",
-    ignoreFocusOut: true,
-  });
-  if (useProject == "Yes") {
-    // Have them pick a project
-    project = await pickProject(api);
-    if (project == undefined) {
-      return;
-    }
-  }
-  // Pick between isfs and isfs-readonly, code and web files
+  // Prompt the user for edit or read-only
   const mode = await vscode.window.showQuickPick(
-    project
-      ? [
-          {
-            value: AccessMode.Code,
-            label: `Edit Code in project '${project}'`,
-            detail: "Edit all documents in the project.",
-          },
-          {
-            value: AccessMode.CodeReadonly,
-            label: `$(lock) View Code in project '${project}'`,
-            detail: "Documents opened from this folder will be read-only.",
-          },
-        ]
-      : [
-          {
-            value: AccessMode.Code,
-            label: `Edit Code in ${namespace}`,
-            detail: "Edit classes, routines and other code assets.",
-          },
-          {
-            value: AccessMode.WebappFiles,
-            label: `Edit Web Application Files for ${namespace}`,
-            detail: "Edit files belonging to web applications that use the namespace.",
-          },
-          {
-            value: AccessMode.CodeReadonly,
-            label: `$(lock) View Code in ${namespace}`,
-            detail: "Documents opened from this folder will be read-only.",
-          },
-          {
-            value: AccessMode.WebappFilesReadonly,
-            label: `$(lock) View Web Application Files for ${namespace}`,
-            detail: "Documents opened from this folder will be read-only.",
-          },
-        ],
-    { placeHolder: "Choose the type of access", ignoreFocusOut: true }
+    [
+      {
+        value: FILESYSTEM_SCHEMA,
+        label: `$(pencil) Edit Code in ${namespace}`,
+        detail: "Documents opened in this folder will be editable.",
+      },
+      {
+        value: FILESYSTEM_READONLY_SCHEMA,
+        label: `$(lock) View Code in ${namespace}`,
+        detail: "Documents opened in this folder will be read-only.",
+      },
+    ],
+    { title: "Choose the type of access", ignoreFocusOut: true }
```

Comment:
```suggestion
    { placeHolder: "Choose the type of access", ignoreFocusOut: true }
```
For consistency with the other steps.